frontend/src/app/pages/exercises/exercise/shared/core/drag-element.service.ts

```diff
@@ -168,7 +169,8 @@ export class DragElementService {
                                         .patientTemplates.length
                             )
                         ]!,
-                        this.transferringTemplate.template.name
+                        this.transferringTemplate.template.name,
+                        MapPosition.create(position)
```

Comment:
You can change the following lines below then:

```ts
                            patient: {
                                ...patient,
                                position,
                            },
```

to

```ts
                            patient,
```

Reply:
I would defer this to #612